### PR TITLE
Autocomplete: update demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 1]
```diff
@@ -8,21 +8,27 @@ Use one of the following properties to supply data to the component:
 Accepts a local data array.
 
 * [dataSource](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#dataSource)            
-Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the "Custom Item Template and Data Source Usage" component that uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the following components:
+
+    - "Custom Item Template and Data Source Usage"    
+    Uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+    
+    - "With Custom Store and Search Options"
```
Comment (arminal) [inline JSDemos/Demos/Autocomplete/Overview/description.md]: with?

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 2]
```diff
@@ -8,21 +8,27 @@ Use one of the following properties to supply data to the component:
...
 
 The [value](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#value) property stores the selected item. You can use the same property to select an item programmatically, as shown in the "Disabled" Autocomplete component.
 
-If the data source contains objects, you should specify the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property. It accepts a data field name that uniquely identifies each data object. In this demo, **valueExpr** is set only for the "Custom Item Template and Data Source Usage" Autocomplete because others are bound to arrays of primitive values.
+If the data source contains objects, you should specify the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property. It accepts a data field name that uniquely identifies each data object. In this demo, [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) is specified only for the "Custom Item Template and Data Source Usage" and "With Custom Store and Search Options" components because others are bound to arrays of primitive values.
```
Comment (arminal) [inline JSDemos/Demos/Autocomplete/Overview/description.md]: Same note: With Custom... With? 

Also, when you say "others are bound to arrays", what are you referring to? other what? clarify.

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 3]
```diff
@@ -8,21 +8,27 @@ Use one of the following properties to supply data to the component:
...
 
 ## Configure Search Parameters
 
 When a user types the first character, Autocomplete displays suggestions. To increase the number of characters that triggers suggestions, use the [minSearchLength](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#minSearchLength) property. You can also specify the time interval Autocomplete should wait before it displays suggestions. Assign this time interval in milliseconds to the [searchTimeout](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchTimeout) property. See the "With Custom Search Options" Autocomplete component for an example.
```
Comment (arminal) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
When a user types the first character, Autocomplete displays suggestions. To increase the number of characters that trigger suggestions, use the [minSearchLength](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#minSearchLength) property. You can also specify the time interval Autocomplete should wait before it displays suggestions. Assign this time interval in milliseconds to the [searchTimeout](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchTimeout) property. See the "With Custom Search Options" Autocomplete component for an example.
```
same question also here about the use of the word "With" in component title

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 4]
```diff
@@ -8,21 +8,27 @@ Use one of the following properties to supply data to the component:
...
+Usually, the data field that supplies Autocomplete with suggestions ([valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr)) is the same data field that is searched for the typed text. If you use two different fields, assign the field that supplies Autocomplete with suggestions to the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property and the field to be searched to the [searchExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchExpr) property. Note that [searchExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchExpr) also accepts arrays if you want to search multiple fields.
```
Comment (arminal) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
In most cases, the data field that supplies Autocomplete with suggestions [([valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr)) - how does this name (valueExpr) fit here? what is it referring to?] is the same data field that is used to search for the entered text. If you use two different fields, assign the field that supplies Autocomplete with suggestions to the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property and the field used to search to the [searchExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchExpr) property. Note that [searchExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchExpr) also accepts arrays if you want to search multiple fields.
```

File: JSDemos/Demos/Autocomplete/Overview/description.md (hunk 1 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Autocomplete/Overview/description.md]: Would "Custom Store and Search Options" be better?

File: JSDemos/Demos/Autocomplete/Overview/description.md (hunk 2 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Autocomplete/Overview/description.md]: I can change it to "other components"

File: JSDemos/Demos/Autocomplete/Overview/description.md (hunk 1 above)
Comment (arminal) [reply JSDemos/Demos/Autocomplete/Overview/description.md]: Absolutely. Please make this edit everywhere else this name appears.

File: JSDemos/Demos/Autocomplete/Overview/description.md (hunk 2 above)
Comment (arminal) [reply JSDemos/Demos/Autocomplete/Overview/description.md]: Yes, other components.

**Pull request merged**